core/domain/feedback_services.py

```diff
@@ -172,38 +244,58 @@ def get_next_page_of_all_feedback_messages(
     return (result_messages, new_urlsafe_start_cursor, more)
 
 def get_thread_analytics_multi(exploration_ids):
-    """Returns a dict with feedback thread analytics for the given exploration
-    ids.
+    """Fetch all FeedbackAnalytics for all the given exploration ids
+
+    Args:
+        exploration_ids: list of str. A list of exploration id
+
+    Returns:
+        list of FeedbackAnalytics
     """
-    return feedback_jobs_continuous.FeedbackAnalyticsAggregator.get_thread_analytics_multi( # pylint: disable=line-too-long
-        exploration_ids)
+    return feedback_jobs_continuous.FeedbackAnalyticsAggregator.\
```

Comment:
That's strange. What is the pylint error? Is your pylint check reading from oppia's .pylintrc?

I ask because Travis-CI runs the pylint tests on all files, and the old version passed fine...